torch/csrc/autograd/variable.cpp

```diff
@@ -79,22 +79,25 @@ namespace impl {
     if (self.is_view()) {
       // NB: is_view() ==> get_autograd_meta()
       auto diff_view_meta = static_cast<DifferentiableViewMeta*>(get_autograd_meta(self));
-      // Do not use handle_view_on_rebase here as check_inplace should have been called before this
-      // and either throw an error or clear the warning
-      TORCH_INTERNAL_ASSERT(diff_view_meta->creation_meta == CreationMeta::DEFAULT);
-      TORCH_INTERNAL_ASSERT(gradient_edge.input_nr == 0);
-      TORCH_INTERNAL_ASSERT(gradient_edge.function);
-      TORCH_CHECK(
-          gradient_edge.function->num_inputs() == 1,
-          "Functions which modify views in-place must return a single Variable");
-      diff_view_meta->output_nr_ = gradient_edge.input_nr;
-      auto copy_slices = std::make_shared<CopySlices>(
-          diff_view_meta->base_, at::TensorGeometry(self), diff_view_meta->view_fn_, std::move(gradient_edge.function));
-      set_gradient_edge(diff_view_meta->base_, {std::move(copy_slices), 0});
-      self.grad_fn(); // trigger an update to the view's grad_fn
-    } else {
-      set_gradient_edge(self, std::move(gradient_edge));
+      if (diff_view_meta->creation_meta != CreationMeta::MULTI_OUTPUT_SAFE) {
```

Comment:
Can you add a comment pointing to the `[ View + Inplace detection ]` Note here and in the same if statement below?
To make it clear for future readers why these are not "real views" for now.